helix-front/tsconfig.json

```diff
@@ -10,6 +14,8 @@
     "experimentalDecorators": true,
     "target": "es2015",
     "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
These two options are the other part of the solution to https://stackoverflow.com/q/49256040/1732222